presto-docs/src/main/sphinx/release/release-314.rst

```diff
@@ -0,0 +1,71 @@
+===========
+Release 314
+===========
+
+General Changes
+---------------
+
+* Fix queued queries counter in UI. (:issue:`894`)
+* Fix incorrect results for ``BETWEEN`` with a ``NULL`` min or max value. (:issue:`877`)
+* Fix query history leak in coordinator. (:issue:`939` :issue:`944`)
+* Improve performance of :func:`json_parse` function. (:issue:`904`)
+* Visualize plan structure in ``EXPLAIN`` output. (:issue:`888`)
+* Add support for positional access to ``ROW`` fields via the subscript
```

Comment:
"constant positional access"

Reply:
"constant positional access" sounds very strange. If anything, it should be "... via the subscript operator using a constant index". But I think that's just a detail (covered in the appropriate documentation section and via error reporting) and not relevant to the release notes.